src/main/java/racingcar/utils/message/ErrorMessage.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.utils.message;
+
+public enum ErrorMessage {
+    INVALID_NAME_CONDITION("이름은 공백 없는 1~5자의 영어 또는 한글이어야 합니다."),
+    INVALID_NAME_NUMBER("자동차의 갯수는 2~5개 사이여야 합니다."),
+    DUPLICATE_CAR_NAME("중복되는 자동차 이름이 존재합니다."),
+    INVALID_NAMES_INPUT("올바른 이름 형식이 아닙니다. ex) jun,pobi,woni"),
+    INVALID_DIGIT_FORM("숫자의 형식이 아닙니다."),
+    INVALID_ROUND_RANGE_INPUT("횟수는 1이상 10이하만 입력할 수 있습니다.");
```

Comment:
enum으로 에러 메시지를 관리하시는데 다음과 같은 방법을 사용해보시면 좋을 것 같습니다.

```suggestion
public enum ErrorMessage {
    INVALID_NAME_CONDITION("이름은 공백 없는 1~5자의 영어 또는 한글이어야 합니다."),
    INVALID_NAME_NUMBER("자동차의 갯수는 2~5개 사이여야 합니다."),
    DUPLICATE_CAR_NAME("중복되는 자동차 이름이 존재합니다."),
    INVALID_NAMES_INPUT("올바른 이름 형식이 아닙니다. ex) jun,pobi,woni"),
    INVALID_DIGIT_FORM("숫자의 형식이 아닙니다."),
    INVALID_ROUND_RANGE_INPUT("횟수는 1이상 10이하만 입력할 수 있습니다.");
    
    public void throwError(){
       throw new IllegalArgumentException(message);
    } 
```
아래는 관련해서 참고 자료입니다 ㅎㅎ
https://github.com/cheese10yun/spring-guide/blob/master/docs/exception-guide.md

Reply:
오... 에러를 바로 던지는 방법은 생각치도 못했네요. 하나 배워갑니다!!

Reply:
저도 이번 미션 진행하면서 알게 된 지식인데 enum을 Interface와 연관 지어 보시는 것도 추천 드립니다. 😄